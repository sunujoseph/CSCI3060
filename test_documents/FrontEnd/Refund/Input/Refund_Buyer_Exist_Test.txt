login
admin1
refund
item1
johnadams
jonnyadamns
200
