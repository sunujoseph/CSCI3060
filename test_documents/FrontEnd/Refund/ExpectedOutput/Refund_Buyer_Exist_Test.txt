Please log in to the system: 
Please enter your username: 
Enter command: 
Enter Item Name: 
Enter buyer username: 
Error: Invaild Username